PuzzleSlider/src/EngineTest/EngineTest.java

```diff
@@ -26,30 +27,24 @@ public EngineOverride( int sz ) throws InvalidArgumentException {
 		
 		public GameData getGameData() { return mGameData; }
 		public boolean moveOverride( GridPoint point ) { return move(point); }
+		public boolean moveOverride(int row, int col) { return move(row, col); }
 		public int manhattanDistance( GridPoint pt1, GridPoint pt2 ) { return getManhattanDistance(pt1, pt2); }
 		public GridPoint findEmptyTileAroundOverride( GridPoint point ) { return findEmptyTileAround(point); }
 		
 	}
-	
-	
-	// --------------------------------
-	// To Test
-	// --------------------------------
-
-	// Maybe, maybe not necessary: public boolean move(int row, int col);
-	// protected boolean move( GridPoint point );
-
-	// --------------------------------
 
 	@Test(expected = InvalidArgumentException.class) 
 	public void invalidConvstructorTest() throws InvalidArgumentException{
 		Engine engine = new Engine(3);
+		engine = new Engine(-1);
```

Comment:
The first call will throw, resulting in the test method finishing and not calling the two following contructors